CHANGES.txt

```diff
@@ -35,6 +35,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       OverrideEnvironment. Enable env.setdefault() method, add tests.
     - Raise an error if an option (not otherwise consumed) is used which
       looks like an abbreviation of one one added by AddOption. (#3653)
+    - Change error (to UserError) and error message in case a requested
```

Comment:
When a Tool() is requested but not found,  a UserError will new be thrown and potentially and error message if requested.  

Lead with Tool...

Reply:
Done.